src/components/Home/AmountInput.js

```diff
@@ -12,20 +12,22 @@ import colors from '../../styles/colors';
 
 export default function AmountInput({ value, error, maxLength, onChange, tintColor, autofocus }) {
 	return (
-		<View style={{ flexDirection: 'row' }}>
+		<View style={{ flexDirection: 'row', justifyContent: 'center' }}>
```

Comment:
Not a big fan of `justifyContent: 'center'`
Can someone else take a decision ?

Reply:
I also use justifyContent with flexbox. Why are you not a big fan? Maybe there's something i don't know about it.

Reply:
Me too ahah, I was just saying that I prefer the amount left aligned than centered